libs/api-client-tiger/README-DEV.md

```diff
@@ -2,17 +2,25 @@
 
 Client is generated with [openapi-generator](https://github.com/OpenAPITools/openapi-generator) using [typescript-axios](https://github.com/OpenAPITools/openapi-generator/tree/master/modules/openapi-generator/src/main/resources/typescript-axios) module, according to GoodData Cloud and GoodData.CN [OpenAPI specification](https://github.com/OAI/OpenAPI-Specification).
 
+It means the client is generated based on code on running backend api and stored and versioned in this repository.
+
 ## How to generate client
 
-`BASE_URL=http://your-tiger-host npm run generate-client`
+1. You need to install Java runtime environment (because openapi-generator is a java based tool).
+
+2. Maybe you will need to add execution priviledges to the script file:
+   `chmod +x ./scripts/generate.js`
```

Comment:
I came across this one too, can you please try commiting the changed mode?

Reply:
Yes. Good idea. I've just amend that. Needed to add some dummy newline to pass `rush lint-staged-files`. GH does not show the diff in privileges.

Reply:
Hm. seems it did not work. I do not see the change in the git log.

Reply:
I tried a separate commit. It seems the change just does not pass trough. I see it staged, but when I commit it, it is not in the log summary. I guess it is removed by some precommit hook.

Reply:
please try to commit with `--no-verify`